course/tests/tests_exportcourse.py

```diff
@@ -0,0 +1,29 @@
+from django.core.management import CommandError, find_commands
+from django.test import TestCase
+from django.core import management
+from io import StringIO
+import sys
+
+
+class CommandTests(TestCase):
+    databases = '__all__'
+    fixtures = ["dumps/courseData.json"]
+
+    def test_command(self):
+        sys.stdout = out = StringIO()
+        management.call_command('exportcourse', 1, stdout=out, verbosity=3)
+        self.assertIn("Exporting course Spanish for English speakers course", out.getvalue())
+
+    def test_calling_a_command_without_argument(self):
+        with self.assertRaisesMessage(CommandError, 'Error: the following arguments are required: course_id'):
+            sys.stdout = out = StringIO()
+            management.call_command('exportcourse', stdout=out)
+
+    def test_exportcourses(self):
+        with self.assertRaisesMessage(CommandError, "Unknown command: 'exportcourses'"):
```

Comment:
but `exportcourses` exists

Reply:
We have `exportcourse` not `exportcourses`

Reply:
since this command actually doesn't exist, we don't need this test case. It's just testing Django's built in functionality. Please remove `test_exportcourses`